h2/src/main/org/h2/mvstore/MVStore.java

```diff
@@ -1603,6 +1607,17 @@ private long getFileLengthInUse() {
         return result;
     }
 
+    /**
+     * Get the index of the first block after last occupied one.
+     * It marks the beginning og the last (infinite) free space.
+     *
+     * @return block index
+     */
+    private long getAfterLastBlock() {
```

Comment:
The return type long should be changed to int.

Reply:
And everything else, of course...